### PR TITLE
First stab at adding a gallery for the grants

### PR DESCRIPTION
**Pull request by nathanshammah**
First stab at adding a gallery for the grants, using Airtable. More fields can be populated for each item from Airtable, possibly not changing this link. 

Address #12 and possibly help solve #77.

### CODE REVIEW

**Comment by nathanshammah** [comment]
The preview is here: https://airtable.com/shrx0ss7J17LTX7cQ

**Review by crazy4pi314 — COMMENTED** [review]
I like the more horizontal line view, I think I would make the height o f the viewport a bit bigger, and once the short descriptions of the projects are on the cards, remove the duplication. This will be great 💖

**Comment by crazy4pi314** [comment]
One other thing, for clarity I would turn off the controls on the embed, but that's just me.

**Comment by willzeng** [comment]
I like having this in an auto-updating format. Makes it easier for us to stay current.

Is there a way to change the cards so that a short form description of the project (we could add a field for this) and important links are visible? I think those fields are more important to be available than the exact timing and team size etc.

File: grants.html [hunk 1]
```diff
@@ -71,7 +71,10 @@
                             <a href="https://unitaryfund.typeform.com/to/j0kAOd" target="_blank" style="background-color:var(--color-yellow);">Apply here</a>.
                         </p>
 
-                        <p class="subtitle leading-arrow">Grants Made</p>
+                        <p class="subtitle leading-arrow">Grants Made: Gallery</p>
+                        <iframe class="airtable-embed" src="https://airtable.com/embed/shrx0ss7J17LTX7cQ?backgroundColor=yellow&layout=card&viewControls=on" frameborder="0" onmousewheel="" width="100%" height="533" style="background: transparent; border: 1px solid #ccc;"></iframe>
```
Comment (nathanshammah) [inline grants.html:75]:
```suggestion
                        <iframe class="airtable-embed" src="https://airtable.com/embed/shrx0ss7J17LTX7cQ?backgroundColor=yellow&layout=card" frameborder="0" onmousewheel="" width="100%" height="533" style="background: transparent; border: 1px solid #ccc;"></iframe>
```

File: grants.html [hunk 2]
```diff
@@ -71,7 +71,10 @@
                             <a href="https://unitaryfund.typeform.com/to/j0kAOd" target="_blank" style="background-color:var(--color-yellow);">Apply here</a>.
                         </p>
 
-                        <p class="subtitle leading-arrow">Grants Made</p>
+                        <p class="subtitle leading-arrow">Grants Made: Gallery</p>
+                        <iframe class="airtable-embed" src="https://airtable.com/embed/shrx0ss7J17LTX7cQ?backgroundColor=yellow&layout=card" frameborder="0" onmousewheel="" width="100%" height="533" style="background: transparent; border: 1px solid #ccc;"></iframe>
```
Comment (nathanshammah) [inline grants.html:75]:
```suggestion
                        <iframe class="airtable-embed" src="https://airtable.com/embed/shrx0ss7J17LTX7cQ?backgroundColor=yellow&layout=card&viewControls=on" frameborder="0" onmousewheel="" width="100%" height="533" style="background: transparent; border: 1px solid #ccc;"></iframe>
```

**Comment by nathanshammah** [comment]
Thanks @crazy4pi314 and @willzeng. While I also do not enjoy the view of the controls, these knobs make the database more interactive, as the list becomes a pivot table. More power to the reader.

 @willzeng yes, this was just a first stab at it. If you have a look now I added for all available items a description and a link. We can add more fields like arXiv etc.

**Comment by crazy4pi314** [comment]
> While I also do not enjoy the view of the controls, these knobs make the database more interactive, as the list becomes a pivot table. More power to the reader.

I guess my thought was that they can open the table in the airtable view (button at the bottom and then its the full window size and folks can use the controls there). If others have strong opinions to keep them that's fine.

**Comment by crazy4pi314** [comment]
I think based on discussions we could close this now till we find a non-airtable solution for the grants?

**Comment by willzeng** [comment]
works for me

**Pull request closed without merging**